modules/index.js

```diff
@@ -0,0 +1,51 @@
+import tslib from '../tslib.js';

```

Comment:
Why not add scripts to copy `tslib.es6.js` to `modules/index.js`?

import cjs from mjs will cause warning in Angular CLI.

Reply:
Just wanna add that I was getting an error around this line in a react application after this update: 

```
index.js:3 Uncaught (in promise) TypeError: Cannot destructure property '__extends' of 'i(...)' as it is undefined.
    at Module.97763 (index.js:3)
    at i (bootstrap:18)
    at sharing:34
    at Object.u.<computed> (remoteEntry.js:1)
    at i (remoteEntry.js:1)
    at Module.8209 (209.2823e81d61a69038a4b4.chunk.js:2)
    at Function.i (remoteEntry.js:1)
```

![image](https://user-images.githubusercontent.com/52320810/95463786-d64c4880-0996-11eb-920e-9fe8a66dafad.png)

Let me know if I shall create an issue for this.

Some specs:
React 16.13.0
Bundled using: Webpack 5

Reply:
@udayrajMT if you could make a sample that'd be great, I have some webpack tests using modules in here but must have missed something 👍🏻

Reply:
Tried running it on a react-start-kit with the upgraded version, but that seems to run fine 🤔 
Will check my configuration again and let you know if I find a way to reproduce the issue in a standalone app